fluid-project/trivet
# chore: lint YAML files

## Description

Adds GitHub Actions YAML files to linting configuration.

## Steps to test

1. Run `npm run lint`.

**Expected behavior:** GitHub Actions YAML files are linted.

## Additional information

Not applicable.

## Related issues

Not applicable.
